src/Data/Filters/UserSiteMatch.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * This file provides an implementation of the UserSiteMatch filter.
+ *
+ * PHP Version 7
+ *
+ * @category   Data
+ * @package    Main
+ * @subpackage Data
+ * @author     Dave MacFarlane <david.macfarlane2@mcgill.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+namespace LORIS\Data\Filters;
+
+/**
+ * UserSiteMatch filters out data for any resource which is not part of one of the
+ * user's sites. An Instance using the UserSiteMatch filter must implement a
```

Comment:
Instance -> DataInstance

Reply:
upon rereading, that didn't make sense either, so I rephrased the sentence a little. Is it better now?